Assignment4.txt

```diff
@@ -0,0 +1,9 @@
+Clarke Littlejohn
+
+1) From what i understand there is not really a difference between the two but from readng online string points to a terminator,
```

Comment:
right - the only real difference is that a string's character array must point to a NULL terminator.